docs/source/specifications.rst

```diff
@@ -105,7 +105,7 @@ Section [app]
     garden_requirements = graph
 
   Please note that if it doesn't work, it might be because of the garden
-  package itself. Refer to the author of the package if he already tested
+  package itself. Refer to the author of the package if they already tested
```

Comment:
i like the singular they, but here there could even be multiple authors, so i think we could even use a plural form.
```suggestion
  package itself. Refer to the author(s) of the package if they already tested
```